cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/PostgresSinkConsumerIT.java

```diff
@@ -0,0 +1,277 @@
+                .get().getValue().getIpAddress();
+        System.out.println("postgres ip " + POSTGRES_IP);
+        System.out.println("Postgres container started.");
+        // createTableInPostgres();
+
+        Connection connection = DriverManager.getConnection(postgreSQLContainer.getJdbcUrl(),
+                postgreSQLContainer.getUsername(),
+                postgreSQLContainer.getPassword());
+
+        ConnectorConfiguration connector = ConnectorConfiguration
+                .forJdbcContainer(postgreSQLContainer)
+                .with("connector.class", "io.confluent.connect.jdbc.JdbcSinkConnector")
+                .with("tasks.max", "1")
+                .with("topics", "dbserver1.public.test_table")
+                .with("database.server.name", "dbserver1")
+                .with("dialect.name", "PostgreSqlDatabaseDialect")
+                .with("table.name.format", "sink")
+                .with("connection.url", "jdbc:postgresql://" + POSTGRES_IP + ":5432/postgres?user=postgres&password=postgres&sslMode=require")
+                .with("auto.create", "true")
+                .with("insert.mode", "upsert")
+                .with("pk.fields", "id")
+                .with("pk.mode", "record_key")
+                .with("delete.enabled", "true")
+                .with("auto.evolve", "true")
+                .with("value.converter", "org.apache.kafka.connect.json.JsonConverter")
+                .with("value.converter.schemas.enable", "true")
+                .with("key.converter", "org.apache.kafka.connect.json.JsonConverter")
+                .with("key.converter.schemas.enable", "true");
+
+        debeziumContainer.registerConnector("my-connector", connector);
+
+        HOST_ADDRESS = InetAddress.getLocalHost().getHostAddress();
+        TestHelper.setHost(HOST_ADDRESS);
+        BOOTSTRAP_SERVER = kafka.getNetworkAliases().get(0) + ":9092";
+        TestHelper.setBootstrapServer(BOOTSTRAP_SERVER);
+        String createTableSql = "CREATE TABLE IF NOT EXISTS test_table (id int primary key, first_name varchar(30), last_name varchar(50), days_worked double precision)";
+        TestHelper.execute(createTableSql);
+        String deleteFromTableSql = "DELETE FROM test_table";
+        TestHelper.execute(deleteFromTableSql);
+
+        // Initialise expected_data.
+
+        for (int i = 0; i < recordsInserted; i++) {
+            Map<String, Object> expected_record = new LinkedHashMap<String, Object>();
+            expected_record.put("id", new Integer(i));
+            expected_record.put("first_name", new String("first_" + i));
+            expected_record.put("last_name", new String("last_" + i));
+            expected_record.put("days_worked", new Double(23.45));
+            expected_data.add(expected_record);
+        }
+
+        cdcContainer = TestHelper.getCdcsdkContainer();
+        cdcContainer.withNetwork(network);
+        cdcContainer.start();
+        System.out.println("CDCSDK container created at: {}"
```

Comment:
Remove the `System.out.println()` from everywhere and use `LOGGER.info()` if and wherever required.